connect/src/core.ts

```
@@ -1,404 +1,97 @@
-      await connectWebSocket(this.url);
-      return this.buildClient();
-    } catch (e) {
-      this.notifyConnectionChange("not_connected");
-      return this.connectToPort();
-    }
-  }
-
-  async findPort(): Promise<number> {
-    const ports = [...Array(10).keys()].map((_, i) => {
-      return checkPort(DEFAULT_PORT + i);
     });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)